distributed/cli/dask_worker.py

```diff
@@ -191,8 +191,11 @@ def del_pid_file():
         t = Worker
 
     if not scheduler and not scheduler_file:
-        raise ValueError("Need to provide scheduler address like\n"
-                         "dask-worker SCHEDULER_ADDRESS:8786")
+        try:
+            scheduler = ":".join((config['scheduler-ip'], config['scheduler-port']))
+        except KeyError:
+            raise ValueError("Need to provide scheduler address like\n"
+            "dask-worker SCHEDULER_ADDRESS:8786 or with environment variables")
```

Comment:
If we're going to list environment variables as an option then we probably need to explain them.  In the interests of brevity I recommend just avoiding this change.